org.eclipse.mylyn-target/org.eclipse.mylyn.target.target

```diff
@@ -30,21 +30,21 @@
 			<unit id="org.eclipse.egit.github.core" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/egit/updates-6.8/"/>
+			<repository location="https://download.eclipse.org/egit/updates-6.9/"/>
 			<unit id="org.eclipse.egit.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.jgit.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.jgit.lfs.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.37.0/"/>
+			<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.38.0/"/>
 			<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/rt/ecf/snapshot/site.p2/3.14.41.v20240114-0141/"/>
+			<repository location="https://download.eclipse.org/rt/ecf/3.14.41/site.p2/3.14.41.v20240114-0141/"/>
 			<unit id="org.eclipse.ecf.remoteservice.sdk.feature.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/cdt/builds/11.5/cdt-11.5.0-rc1/"/>
+			<repository location="https://download.eclipse.org/tools/cdt/builds/11.6/cdt-11.6.0-rc1/"/>
```

Comment:
This is the latest contribution.  And this is the latest release.

https://download.eclipse.org/tools/cdt/releases/11.5/

It's odd to build against an old RC1.